### PR TITLE
[public-api] Add test for CreateToken when feature flag enabled

### PR DESCRIPTION
**Pull request by easyCZ**
## Description

Ensures we have coverage for both cases of the feature flag. Currently returns unimplemented, but will be extended in future PRs.

## Related Issue(s)

Fixes #

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

File: components/common-go/experiments/flags.go [hunk 1]
```diff
@@ -6,11 +6,15 @@ package experiments
 
 import "context"
 
+const (
+	PersonalAccessTokensEnabledFlag = "personalAccessTokensEnabled"
+)
+
 // IsMyFirstFeatureFlagEnabled example usage of a flag
 func IsMyFirstFeatureFlagEnabled(ctx context.Context, client Client, attributes Attributes) bool {
 	return client.GetBoolValue(ctx, "isMyFirstFeatureEnabled", false, attributes)
```
Comment (andrew-farries) [inline components/common-go/experiments/flags.go:15]: Do you want a constant for this experiment name too?

File: components/public-api-server/pkg/apiv1/tokens_test.go [hunk 2]
```diff
@@ -12,24 +12,31 @@ import (
...
 	protocol "github.com/gitpod-io/gitpod/gitpod-protocol"
 	"github.com/gitpod-io/gitpod/public-api-server/pkg/auth"
 	"github.com/golang/mock/gomock"
-	"github.com/google/uuid"
 	"github.com/stretchr/testify/require"
 )
 
 func TestTokensService_CreatePersonalAccessTokenWithoutFeatureFlag(t *testing.T) {
-	t.Run("returns a personal access token", func(t *testing.T) {
-		serverMock, client := setupTokensService(t)
```
Comment (andrew-farries) [inline components/public-api-server/pkg/apiv1/tokens_test.go:25]: Can remove this blank line.

**Review by andrew-farries — APPROVED** [review]
/hold

File: components/common-go/experiments/flags.go (hunk 1 above)
Comment (easyCZ) [reply components/common-go/experiments/flags.go:15]: I'll remove that flag all together. Was there mostly as an example

**Review by sagor999 — APPROVED** [review]

**Pull request merged**